test/src/820-https_s3/main

```diff
@@ -8,24 +8,21 @@ cvmfs_test_name="HTTPS S3 implementation"
 cvmfs_test_autofs_on_startup=false
 cvmfs_test_suites="quick"
 
-set -x
-
 SETTING_DIR="/tmp/cvmfs_testing_out"
```

Comment:
This should become the current working directory in the `main()` function as the canonical place to store test case output, config, etc.